docs/src/add-services/mysql/_index.md

```diff
@@ -201,24 +201,49 @@ If your database relationship has a password, pass the `-p` switch and enter the
+With version `10.0` or later, you can define multiple users with different permissions for your database.
 To do so, define multiple endpoints in your [service configuration](#configuration-options).
 
 For each endpoint you add, you can define the following properties:
 
 | Name             | Type                     | Required | Description |
 | ---------------- | ------------------------ | -------- | ----------- |
-| `default_schema` | `string`                 |          | Which of the schemas defined above to default to. If not specified, the `path` property of the relationship is `null` and so tools such as the Platform CLI can't access the relationship. |
+| `default_schema` | `string`                 |          | Which of the defined schemas to default to. If not specified, the `path` property of the relationship is `null` and so tools such as the Platform CLI can't access the relationship. |
 | `privileges`     | A permissions dictionary |          | For each of the defined schemas, what permissions the given endpoint has. |
 
 Possible permissions:
 
-* `ro`: Only SELECT queries are allowed.
-* `rw`: SELECT queries and INSERT/UPDATE/DELETE queries are allowed.
-* `admin`: All queries are allowed including data definition language (DDL) queries (such as `CREATE TABLE`, `DROP TABLE`).
+| Name        | Type          | Description                                         |
+| ----------- | ------------- | --------------------------------------------------- |
+| Read-only   | `ro`          | Can select, create temporary tables, and see views. |
+| Read-write  | `rw`          | In addition to read-only permissions, can also insert, update, delete, manage and execute events, execute routines, create and drop indexes, manage and execute triggers, and lock tables. |
+| Admin       | `admin`       | In addition to read-write permissions, can also create, drop, and alter tables; create views; and create and alter routines. |
+| Replication | `replication` | For [replicating databases](./mysql-replication.md). In addition to read-only permissions, can also lock tables. |
+
+You can also specify `schemas` for [multiple databases](#multiple-databases).
+If neither `schemas` nor `endpoints` is included, it's equivalent to the following default:
+
+```yaml {location=".platform/services.yaml"}
+    configuration:
+        schemas:
+            - main
+        endpoints:
+            mysql:
+                default_schema: main
+                privileges:
+                    main: admin
+```
+
+If either `schemas` or `endpoints` are defined, no default is applied and you have to specify the full configuration.
+
+## Multiple databases
+
+With version `10.0` or later, you can define multiple databases.
+To do so, define multiple `schemas` in your [service configuration](#configuration-options).
 
+You can also specify multiple `endpoints` for [permissions](#define-permissions).
```

Comment:
:bulb: :arrow_forward:
Would it make sense to reintroduce the concept of "users" in that context, since this is what endpoints are.
Or is the link back to permissions clear enough?

:hammer_and_wrench:
Maybe that could be enough?
```suggestion
You can also specify multiple user `endpoints` for [permissions](#define-permissions).
```

Reply:
Here, I'd say it doesn't add so much value as concept doesn't help me see what needs to be defined here. I feel like endpoints are actually endpoints and it's not necessarily important if they're users or not. Sometimes can be, but for that the link would be enough.